libs/Einsums/Runtime/src/Runtime.cpp

```diff
@@ -65,7 +65,16 @@ EINSUMS_EXPORT BOOL WINAPI termination_handler(DWORD ctrl_type) {
 
 #else
 [[noreturn]] EINSUMS_EXPORT void termination_handler(int signum) {
-    if (signum != SIGINT && runtime_config().einsums.attach_debugger) {
+    bool attach = true;
+
+    try {
+        auto &global_config = GlobalConfigMap::get_singleton();
+        attach = global_config.get_bool("attach-debugger");
+    } catch(...) {
+        attach = true;
+    }
```

Comment:
What if the `get_*` functions accepted a default value? The users wouldn't need to check for exceptions.

```
attach = global_config.get_bool("attach-debugger", true);
```

Reply:
Good idea. I'll implement that really quickly. I want to keep this exception handling here, though, since it occurs in a signal handler. I wouldn't want a terminate signal to raise an uncaught exception on accident.

Reply:
Fair point.